Can't get my AC2889/10 to respond

MartinJRS opened the pull request:
Hello, just installed your plugin. Unfortunately my device is not really responding. Hope it helps to figure out the problem when posting relevant lines from my protocol:

[16/09/2022, 14:59:04] [Philips AC2889/10] Initializing philipsAir platform...
[16/09/2022, 14:59:05] [Philips AC2889/10] [Philips AC2889/10] Initializing accessory...
[16/09/2022, 14:59:05] [Philips AC2889/10] [Philips AC2889/10] Unable to load firmware info: Error: connect ECONNREFUSED 192.168.178.163:80
[16/09/2022, 14:59:05] [Philips AC2889/10] [Philips AC2889/10] Unable to load status info: Error: connect ECONNREFUSED 192.168.178.163:80
[16/09/2022, 14:59:06] [Philips AC2889/10] [Philips AC2889/10] Unable to load filter info: Error: connect ECONNREFUSED 192.168.178.163:80
[16.09.2022, 15:00:05] [Philips Air] [Philips AC2889/10] Unable to get data for polling: Error: spawnSync python3 ETIMEDOUT.
[16.09.2022, 15:00:05] If your have "Error: spawnSync python3 ETIMEDOUT" your need unplug the accessory from outlet for 10 seconds and plug again.

Thanks in advanced, Martin

Sunoo committed:
Minor tweaks to be able to publish alongside main version.

Sunoo committed:
Update README.md

Sunoo committed:
Split out philips-air npm package.

Sunoo committed:
Tweaked dependencies list.

Sunoo committed:
Fix issue loading key.

Sunoo committed:
Please switch to homebridge-philips-air.

Sunoo committed:
Added npm badges

MartinJRS commented:
I have now switched to COAP protocol and get this message in:

Traceback (most recent call last):
  File "/usr/local/lib/node_modules/homebridge-philips-air/node_modules/philips-air/pyaircontrol.py", line 8, in <module>
    from pyairctrl.coap_client import CoAPAirClient
ModuleNotFoundError: No module named 'pyairctrl'
[16/09/2022, 15:14:49] [Philips AC2889/10] [Philips AC2889/10] Unable to load filter info: Error: Command failed: python3 /usr/local/lib/node_modules/homebridge-philips-air/node_modules/philips-air/pyaircontrol.py --ipaddr 192.168.178.163 --protocol coap --status
Traceback (most recent call last):
  File "/usr/local/lib/node_modules/homebridge-philips-air/node_modules/philips-air/pyaircontrol.py", line 8, in <module>
    from pyairctrl.coap_client import CoAPAirClient
ModuleNotFoundError: No module named 'pyairctrl'

MartinJRS commented:
hmmm, is there anybody out there?

MartinJRS commented:
Seems like there is no more support for this plugin ...

MartinJRS commented:
So ... this plugin is dead?

Nivlac7 commented:
How did you install your homebridge plugin? Through the hombridge UI? If not you may have forgotten to install py-air-control package.

The 2889/10 is normally supported (saw on reddit it was working fine for other)

MartinJRS commented:
Thanks for responding.

Yeah, I have installed through the HomeBridge UI. Uhm ... can I find some sort of instructions of how to get all done as required?

Thanks

Nivlac7 commented:
Alright, just did it, nothing complex but that's not fully done through Homebridge UI.

You have to install pithon py-air-control and update the CoAPthon3 dependency manually.
So, in console (terminal in homebridge):

> sudo apt install python3-pip git

> sudo pip3 install py-air-control

> sudo pip3 install -U git+https://github.com/rgerganov/CoAPthon3

You need to put COAP in the settings and the IP of the purifier.

Only issue I have for now is that I cannot power it ON through homekit.

MartinJRS commented:
Alright, thanks many times. Will try it out today ... will let you know.

MartinJRS commented:
Can you please specify these steps?

1st: update the CoAPthon3 dependency manually
2nd: put COAP in the settings and the IP of the purifier

Appologies, I am a total noob in these things.

MartinJRS commented:
Alright, I have now added the device's IP and switched to CoAP ... sorry.

But I don't know what to do with "update the CoAPthon3 dependency manually"

Nivlac7 commented:
You simply have to run the 3 commands I gave you in terminal

Le 28 sept. 2022 à 16:28, Martin ***@***.***> a écrit :

﻿

Alright, I have now added the device's IP and switched to CoAP ... sorry.

But I don't know what to do with "update the CoAPthon3 dependency manually"

—
Reply to this email directly, view it on GitHub<https://github.com/NikDevx/homebridge-philips-air/pull/183#issuecomment-1261000332>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AJJHDPB54FVWJQSLYRVDG4DWARI2PANCNFSM6AAAAAAQOKGB64>.
You are receiving this because you commented.Message ID: ***@***.***>

MartinJRS commented:
Okay, thank you very much. I have now done as described. Seems to work sort of ... when I toggle the Philips via HomeKit, it does not react always ... also the Homebridge Protocol prompts this message:

[AC2889] Unable to get data for polling: Error: spawnSync python3 ETIMEDOUT.

Did I miss something?

qubiack commented:
I had a similar problem at the beginning so I went through all steps described by @Nivlac7. Right now it still doesn't work but I got some other messages:
![image](https://user-images.githubusercontent.com/19862364/200941354-189fb8b4-c848-4816-b30d-3e40ce01708c.png)
Any ideas?

My config:
```
        {
            "devices": [
                {
                    "name": "Bedroom Philips Purifier",
                    "ip": "192.168.50.248",
                    "protocol": "coap"
                }
            ],
            "name": "Philips Air",
            "platform": "philipsAir"
        }
```

ivokik commented:
have exactly the same issue , is this plug in rally dead?
